spec.bs

```diff
@@ -451,8 +451,10 @@ controls which [=origins=] are valid [=aggregation coordinators=]. Every
 <dfn>Default aggregation coordinator</dfn> is an [=aggregation coordinator=]
 that controls which is used for a report if none is explicitly selected.
 
-<dfn>Maximum report contributions</dfn> is a positive integer that controls how
-many contributions can be present in a single report.
+<dfn>Maximum report contributions</dfn> is a [=map=] from [=context type=] to
+positive {{unsigned long}} values. Semantically, it defines the maximum number
```

Comment:
Let's just say "positive integer" -- specs don't usually use WebIDL types except in actual interfaces.